src/zaim.ts

```diff
@@ -20,9 +20,8 @@ type ItemType = "payment" | "income" | "transfer";
 export default class Zaim {
   consumerKey: string;
   consumerSecret: string;
-  token: string;
-  secret: string;
-  callback: RequestCallbackFunction;
+  token!: string;
```

Comment:
Os sorry what `!` means here? 👀

Reply:
`!:` means that the variable has a value, not undefined, but it can be initialized outside of the constructor. See: https://github.com/Microsoft/TypeScript/pull/20075
In this code, `token` and `secret` should be assigned when other functions like `_httpGet` is called, which means their type should be `string`, not `string | undefined`.  
However, they need not be initialized in the constructor. When you don't have access tokens, first you must take the authorization step, and then you can set tokens via `setAccessToken` and `setAccessTokenSecret`.